content/en/apps/concepts/navigation.md

```diff
@@ -29,12 +29,15 @@ Tap the menu icon in the upper right corner of the header to access other pages,
 - **Report Bug**: Let us know if something isn’t working or you encounter errors
 - **Log Out**: Sign out of the app
 
-## Sync Status
+### Sync Status
 
 {{< figure src="sync-status.png" link="sync-status.png" class="right col-6 col-lg-3" >}}
 
-At the bottom of the menu is a notification which provides important about sync status. It lets you know whether reports are being submitted.
+Data synchronisation is important for [offline users]({{ ref="users" }}). These users keep a copy of the data they have access to on their device. They can work from their device while disconnected from the internet (offline), by reading from and writing to their copy of the data. "Sync" (synchronisation) is when data on the device is made to match the data on the server. Synchronisation consists of uploading all new or altered data on the device, and downloading new or altered data from the server. This may include the download of software updates when available.
```

Comment:
This is a great addition! Minor thing to fix would be having the full reference to the linked page to avoid ambiguous references: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#avoid-broken-links

Also, the ref shortcode is missing the `<` & `>`, should be  `{{< ref="full/path" >}})`